modules/shared.py

```diff
@@ -125,6 +125,8 @@ def str2bool(v):
 parser.add_argument('--n-gpu-layers', type=int, default=0, help='Number of layers to offload to the GPU.')
 parser.add_argument('--n_ctx', type=int, default=2048, help='Size of the prompt context.')
 parser.add_argument('--llama_cpp_seed', type=int, default=0, help='Seed for llama-cpp models. Default 0 (random)')
+parser.add_argument('--rope-freq-scale', type=float, default=1.0, help='Size of the prompt context.')
+parser.add_argument('--rope-freq-base', type=int, default=10000, help='Size of the prompt context.')
```

Comment:
Should the default not be 2048?

Reply:
I forgot to change the help text. The actual values correspond to the ones in https://github.com/ggerganov/llama.cpp/pull/2054 and also to the default output of llama-cpp when loading a model in text-generation-webui.